plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/DeltaLakeQueryRunner.java

```diff
@@ -88,12 +88,13 @@ public static DistributedQueryRunner createDeltaLakeQueryRunner(Map<String, Stri
     public static DistributedQueryRunner createS3DeltaLakeQueryRunner(String catalogName, String schemaName, Map<String, String> connectorProperties, String minioAddress, TestingHadoop testingHadoop)
             throws Exception
     {
-        return createS3DeltaLakeQueryRunner(catalogName, schemaName, ImmutableMap.of(), connectorProperties, minioAddress, testingHadoop, queryRunner -> {});
+        return createS3DeltaLakeQueryRunner(catalogName, schemaName, ImmutableMap.of(), ImmutableMap.of(), connectorProperties, minioAddress, testingHadoop, queryRunner -> {});
     }
 
     public static DistributedQueryRunner createS3DeltaLakeQueryRunner(
             String catalogName,
             String schemaName,
+            Map<String, String> coordinatorProperties,
```

Comment:
after extraProperties since it overrides them?

see ordering in https://github.com/trinodb/trino/blob/65559c53263316483718c42e92d3b3b54aff0ce5/testing/trino-testing/src/main/java/io/trino/testing/DistributedQueryRunner.java#L634-L635